src/app/modules/main/chat_section/module.nim

```diff
@@ -534,7 +530,18 @@ method activeItemSet*(self: Module, itemId: string) =
   if self.controller.isCommunity():
     let community = self.controller.getMyCommunity()
     if not community.isPrivilegedUser:
-      self.controller.asyncCheckChannelPermissions(mySectionId, activeChatId)
+      if not chat_item.canView or not chat_item.canPost:
```

Comment:
Are you sure this should be an `or`?

Reply:
Yes. It's an `or` because let's say you have a view only channel, `canView` will be `true`, but `canPost` will be false.
We need to check the permissions in that case so that we can tell the user which token they are missing (in the input bar)